var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -133,7 +133,7 @@ class Sundials(CMakePackage):
     # Options added after v2.7.0
     conflicts('+cuda',          when='@:2.7.0')
     conflicts('+raja',          when='@:2.7.0')
-    conflicts('~int64',         when='@:2.7.0')
```

Comment:
i believe this was correct, prior to 2.7.0 you could NOT install sundials without `+int64`.

Reply:
You are right. The reason I changed it (without actually checking) was that the spec `sundials@2.7.0` failed when I was running some tests. Is there a way to change the default to be `+int64` for '@:2.7.0' and `~int64` otherwise?

Reply:
I don’t think it’s possible yet